tests/cli/main.cpp

```diff
@@ -196,253 +196,188 @@ class client_connection
    std::string wallet_filename;
 };
 
+///////////////////////////////
+// Cli Wallet Fixture
+///////////////////////////////
+
+struct cli_fixture
+{
+    int server_port_number {0};
+    fc::temp_directory app_dir = graphene::utilities::temp_directory_path();
+    std::shared_ptr<graphene::app::application> app1 = start_application(app_dir, server_port_number);
+    client_connection* con = new client_connection(app1, app_dir, server_port_number);
+    std::vector<std::string> nathan_keys { "5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3" };
+
+    std::vector<signed_transaction> import_txs;
+    account_object nathan_acct_before_upgrade;
+    signed_transaction upgrade_tx;
+    account_object nathan_acct_after_upgrade;
+    graphene::wallet::brain_key_info bki;
+    signed_transaction create_acct_tx;
+    signed_transaction transfer_tx;
```

Comment:
Should not define so many different transaction objects in `cli_fixture`; `nathan_acct_before_upgrade` and `nathan_acct_after_upgrade` should not be here as well; they should be declared in individual test cases.